pkg/schema/tracker.go

```diff
@@ -54,9 +55,19 @@ var (
 
 // Tracker is used to track schema locally.
 type Tracker struct {
-	store kv.Storage
-	dom   *domain.Domain
-	se    session.Session
+	store     kv.Storage
+	dom       *domain.Domain
+	se        session.Session
+	toIndexes map[string]map[string]*ToIndexes
+}
+
+// ToIndexes is downstream pk/uk info.
```

Comment:
Maybe we can give `pk/uk info` a more appropriate name.

Reply:
The name is changed to 'downstreamTrack'

Reply:
yes, i think it can be 'downstreamTracker', and i will change it to be a struct which contains createTableStmt and index cache.